cmake/scripts/linux/platform/X11.cmake

```diff
@@ -0,0 +1,8 @@
+core_require_dep(OpenGl)
+core_require_dep(EGL)
+core_require_dep(X)
+core_require_dep(XRandR)
+core_require_dep(LibDRM)
+core_optional_dep(VAAPI)
+core_optional_dep(VDPAU)
+core_optional_dep(OpenGLES)
```

Comment:
can we replace those with 2 vars, like in main CMakeLists.txt?
e.g. PLATFORM_REQUIRED_DEPS and PLATFORM_OPTIONAL_DEPS?